inspirehep/records/api/jobs.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from ...pidstore.api import PidStoreJobs
+from .base import InspireRecord
+
+
+class JobsRecord(InspireRecord):
+    """Authors Record."""
+
+    pid_type = "job"
+
+    es_serializer = "RecordMetadataSchemaJSONV1"
```

Comment:
This is wrong, we should add ``None`` for the moment to be explicit
```suggestion
    es_serializer = None
```